ImputationPipeline/Imputation.wdl

```diff
@@ -536,3 +579,135 @@ task MergeSingleSampleVcfs {
     File output_vcf_index = "~{output_vcf_basename}.vcf.gz.tbi"
   }
 }
+
+task CountSamples {
+	input {
+		File vcf
+	}
+
+	Int disk_size = 100 + ceil(size(vcf, "GB"))
+
+	command <<<
+		bcftools query -l ~{vcf} | wc -l >> nSamples.txt
+	>>>
+
+	runtime {
+		docker: "biocontainers/bcftools:v1.9-1-deb_cv1"
```

Comment:
Once I push the containers we're switching this to the gcr image, correct?

Reply:
yup!